### PR TITLE
Use consistent docstrings for test files

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

Closes https://github.com/GenericMappingTools/pygmt/issues/2571.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: pygmt/tests/test_blockm.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-Tests for blockmean and blockmode.
+Test blockmean and blockmode.
```
Comment (maxrjones) [inline pygmt/tests/test_blockm.py:2]:
```suggestion
Test pygmt.blockmean and pygmt.blockmode.
```

**Review by maxrjones — APPROVED** [review]

File: pygmt/tests/test_psconvert.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 """
-Tests psconvert.
+Tests Figure.psconvert.
```
Comment (yvonnefroehlich) [inline pygmt/tests/test_psconvert.py:2]:
```suggestion
Test Figure.psconvert.
```

File: pygmt/tests/test_histogram.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # pylint: disable=redefined-outer-name
 """
-Tests histogram.
+Test Figure.histogram.
```
[the remark this answers is not in the document]
Comment (seisman) [reply pygmt/tests/test_histogram.py:3]: Done in 21a3c9ef43c7127d8838b4551ac8e6639eb20830.

**Comment by seisman** [comment]
> I am wondering whether we also should update these docstrings:
> 
> ```
> $ grep "Tests" *
> grep: baseline: Is a directory
> grep: data: Is a directory
> test_clib_loading.py:# Tests for load_libgmt
> test_clib_loading.py:# Tests for clib_full_names
> test_helpers.py:Tests the helper functions/classes/etc used in wrapping GMT.
> test_io.py:Tests for input/output (I/O) utilities.
> ```

Done in 09fb8bf4e4cf93aa54b2b76b6bf70bfcd1cbba1e.

**Review by yvonnefroehlich — APPROVED** [review]

**Review by michaelgrund — APPROVED** [review]

**Pull request merged**